### DISCUSSION TITLE
Should i learn one framework but deeply and then just move on to the next framework in cyber or learn a lot of frameworks but just a bit for each one of them??

### DISCUSSION THREAD

**Discussion by EliranDev** [rapid7/metasploit-framework#14739] [Q&A] [answered]
Hi!
I am learning cyber hacking for my own and I have a question.
**Should i learn just one framework (metasploit for example) but really well and then learn other frameworks, or learn just one framework but not getting deep with it, And then just move on to other framework?**
Thank you very much!

**Comment by gwillcox-r7** [comment, marked as answer]
This is a location for discussion requests for new features, not for general discussions, which are better held on our Slack, which you can join via https://join.slack.com/t/metasploit/shared_invite/zt-juutactu-xHoKE3ii7vdmuleDIXfaAw
